test/CompressedSparseRow.hpp

```diff
@@ -0,0 +1,94 @@
+/****************************************************************************
+ * Copyright (c) 2012-2019 by the ArborX authors                            *
+ * All rights reserved.                                                     *
+ *                                                                          *
+ * This file is part of the ArborX library. ArborX is                       *
+ * distributed under a BSD 3-clause license. For the licensing terms see    *
+ * the LICENSE file in the top-level directory.                             *
+ *                                                                          *
+ * SPDX-License-Identifier: BSD-3-Clause                                    *
+ ****************************************************************************/
+
+#ifndef ARBORX_COMPRESSED_SPARSE_ROW_HPP
+#define ARBORX_COMPRESSED_SPARSE_ROW_HPP
+
+#include "VectorOfTuples.hpp"
+
+namespace ext
+{
+
+// Extension for compile-time sequence of integers that defines a helper class
+// template make_index_sequence_with_offset to simplify the creation of
+// std::index_sequence type with O, O+1, O+2, O+N-1 as Ints.
```

Comment:
Can you please use the same name in the comments as in the code?

Reply:
Done